code/game/machinery/computer/law.dm

```diff
@@ -36,9 +47,46 @@
 		if(!atoms_share_level(T, src))
```

Comment:
the `if(istype(O, /obj/item/aiModule))` on L42 needs to be a guard clause

Reply:
![image](https://user-images.githubusercontent.com/77684085/233881681-ab4b6f96-5a16-4979-9310-06e5df0fa72e.png)
Do it like this, returning if its not aiModule then removing one TAB from everything that was inside of the original "if".